src/templates/bq_client.py

```diff
@@ -6,16 +6,19 @@
 import logging
 
 # Replace path to ignore packages that comes in "spark-bigquery-support"
-import sys
-
-logging.info(f"original path: {sys.path}")
-new_path = [p for p in sys.path if not "spark-bigquery-support" in p]
-sys.path = new_path
-logging.info(f"new path: {sys.path}")
+# import sys
```

Comment:
Do we need to uncomment this or delete completely? If we can delete it, we probably need to delete the comment as well.

Reply:
Yes, previously we have to have this section but with the new docker image this can go away.